test/cluster/test_tablets.py

```diff
@@ -1088,6 +1088,47 @@ async def test_tablet_split_finalization_with_migrations(manager: ManagerClient)
     logger.info("Waiting for migrations to complete")
     await log.wait_for("Tablet load balancer did not make any plan", from_mark=migration_mark)
 
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_tablet_split_finalization_with_repair(manager: ManagerClient):
+    injection = "handle_tablet_resize_finalization_wait"
+    cfg = {
+        'enable_user_defined_functions': False, 'enable_tablets': True,
```

Comment:
nit: why do you need to disable `enable_user_defined_functions`, seems completely irrelevant for this test.